code/models/donation.js

```diff
@@ -0,0 +1,25 @@
+// donation mongoose model
+// might need to add a new category to determine if this is a past donation or a suggestions, according to thomas on google docs
+
+// Used https://stackoverflow.com/questions/33934072/requiring-model-schemas-in-another-model-for-mongoose-in-different-files as a reference
+const mongoose = require('mongoose');
+const Location = require('./location');
+const LocationSchema = mongoose.model('Location').schema;
+
+const donationSchema = new mongoose.Schema({
+  itemName: {type: String, required: true},
+  rating: {type: Number, required: true},
+  locationID: {type: LocationSchema, required: true},
+  tripID: {type:String, required:true},
+  category: {type:String, enum:['Health', 'Household', 'Clothing', 'Education', 'Art', 'Sports', 'Miscellaneous', 'Animal Welfare', 'Environment', 'Food']},  
```

Comment:
It might be beneficial to move this enum array into a variable on its own. That way this line doesn't get too long.

Reply:
Okay, I've made the change